.github/workflows/ftc-require-issue.yml

```
@@ -0,0 +1,110 @@
+              for pattern in patterns
+              for match in pattern.findall(text)
+          }
+          if any(is_repo_issue(number) for number in sorted(candidates)):
+              sys.exit(0)
+
+          if (pr.get("state") or "").upper() == "CLOSED":
+              sys.exit(0)
+
+          comment = f"""Thanks for the pull request.
+
+First-time contributors need an associated issue before we can review a PR.
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)